excel/src/main/java/org/apache/metamodel/excel/ExcelUtils.java

```diff
@@ -232,53 +249,119 @@ public static String getCellValue(Workbook wb, Cell cell) {
             throw new IllegalStateException("Unknown cell type: " + cell.getCellType());
         }
 
-        logger.debug("cell {} resolved to value: {}", cellCoordinate, result);
+        logger.debug("cell ({},{}) resolved to value: {}", cell.getRowIndex(), cell.getColumnIndex(), result);
 
         return result;
     }
 
-    private static String getFormulaCellValue(Workbook wb, Cell cell) {
+    private static Object getErrorResult(final Cell cell) {
+        String errorResult;
```

Comment:
You don't need a variable to put the "errorResult" in, you can instead just return it when you get it.

Reply:
sure